06-llm-pr-review/backend/main.py

```
@@ -34,38 +34,33 @@ async def lifespan(app: FastAPI):
 
 
 @app.post("/products/", response_model=ProductDTO)
-async def create_product(
-    payload: ProductCreate,
-    db: AsyncSession = Depends(get_db)
-):
+async def create_product(payload: ProductCreate, db: AsyncSession = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)